.gitmodules

```diff
@@ -2,6 +2,7 @@
         path = lib/ivykis
         url = https://github.com/bazsi/ivykis.git
         ignore = dirty
+	branch = iv-work-pool-support-for-slave-work-items
```

Comment:
Let's use spaces consistently in this file.

Reply:
funny, it's made by 

`
git config -f .gitmodules submodule."lib/ivykis".branch iv-work-pool-support-for-slave-work-items
`

I'll update manually than

Reply:
Hm... All of my git configs use tabs instead of spaces. This file was probably edited manually then :(